### ISSUE TITLE
Export specific datasets from a database/part of a database

### ISSUE THREAD

**Issue by KarinTreyer** [brightway-lca/brightway2-io#70]
bw2io.export of part of a database (e.g. selected activities from ecoinvent) is not possible. It's only possible to export the full db. However, sometimes you want to work on specific activities and change things there, without touching the original datasets and database.

**Comment by BenPortner** [comment]
Hi Karin,

which exporter are you using? `write_lci_excel` has an argument `objs` that allows you to pass a list of activities, which will then be exported. For `write_lci_csv`, this argument is not exposed so you have to use the following workaround:

```python
from bw2io.export.csv import CSVFormatter, safe_filename
import os
import csv

acts = [your activities here]
data = CSVFormatter(database_name, objs=acts).get_formatted_data(None)

safe_name = safe_filename(database_name, False)
filepath = os.path.join(bw.projects.output_dir, "lci-" + safe_name + ".csv")

with open(filepath, "w") as f:
    writer = csv.writer(f)
    for line in data:
        writer.writerow(line)
```

**Comment by cmutel** [comment]
Should be fixed by #72
